zubhub_frontend/zubhub/src/assets/js/styles/views/projects/projectsStyles.js

```diff
@@ -108,20 +137,44 @@ const styles = theme => ({
   },
   heroMessagePrimaryStyle: {
     fontSize: '2.2rem',
-    fontWeight: 1000,
+    fontWeight: '1000',
   },
   MessagePrimaryStyle: {
     fontSize: '2.5rem',
     fontWeight: 1000,
   },
-
+  heroMainMessageSecondaryStyle: {
+    fontSize: '4.5rem',
+    fontWeight: '700',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '2.5rem'
+    },
+  },
+  heroMainMessagePrimaryStyle: {
+    fontSize: '2.2rem',
+    fontWeight: 500,
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '1.2rem'
+    },
+  },
   heroButtonStyle: {
     textTransform: 'none',
-    padding: '0 1em',
-    fontSize: '1.2rem',
+    fontSize: '1.456rem',
     fontWeight: 700,
     marginRight: '1em',
     marginTop: '1.2em',
+    width: 'fit-content',
+    height: '3.95rem',
+    padding: '1.04rem 2.08rem 1.04rem 2.08rem',
```

Comment:
So there are two decimal places here. Why do we need such accuracy?

Reply:
I use the Figma design to show me the exact styles applied, and those are in `px`. I convert them to `rem` because it is more flexible in terms of adjusting according to screen sizes. Conversion from `px` to `rem` gives these decimal values.

Reply:
@aqsaaqeel Can we round the double digit values to single digit at the very minimum? You can eyeball them to see if they align. Usually, there should be minimum need to do such "fine tuning". The figma designs are not supposed to be accurate representations to the exact location, but just indication of how things are aligned. Another benefit of keeping simple numbers is that we don't have to do a lot of fine tuning in other places (containers/padding) - since they will also align to single values.

Reply:
Alright. I'll make the changes!